frontend/bundles/brightspot-training-bundle-default/styleguide/page/list/PageListY.hbs

```diff
@@ -0,0 +1,21 @@
+{{#if items}}
+<div class="PageListY" data-module {{include "includes/PageList-attributes.hbs" this}}>
+    {{include "includes/PageList-header.hbs" this}}
+
+    {{#with items}}
```

Comment:
why the with is defined here if the items are conditioned on the first line?

Reply:
You are 100% right. We could have absolutely just done `{{#each items}}` on line 7, since we have that `{{#if}}` statement. I think what happened is that we had `{{#with items}}` first, and then a developer updated the module to render absolutely nothing with the `{{#if items}}` if there were no items, and did not remove line 5